dredd/dredd_hook.py

```
@@ -54,6 +56,71 @@ def print(*args, **kwargs):
+        if e.code != 404:
+            print('Unexpected HTTP {0} checking series {1}, skipping seed.'.format(e.code, slug))
+            return
+    except URLError as e:
+        print('Could not reach server to check series {0}: {1}.'.format(slug, e))
+        return
+
+    # Add the series via API (queued async)
+    data = json.dumps({'id': {'tvdb': 301824}})
+    if hasattr(data, 'encode'):
+        data = data.encode('utf-8')
+    req = urllib_request.Request(base + '/series', data=data, headers=headers, method='POST')
```

Ruff (0.14.14)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)